cmd/uplink/cmd/cat.go

```diff
@@ -38,9 +38,14 @@ func catMain(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	if u0.Host == "" {
+	if u0.Bucket() == "" {
```

Comment:
Shouldn't this check whether the path is remote?

Reply:
Changed it everywhere in all commands.